build/tools/AutoUpdater/GitHubRunnersCachedImagesCheckCommand.cs

```diff
@@ -69,6 +69,7 @@ public int OnExecute(CommandLineApplication app, IConsole console)
             var forkAccountName = Environment.GetEnvironmentVariable("FORK_ACCOUNT_NAME");
             var accessToken = Environment.GetEnvironmentVariable("AUTOUPDATE_PAT");
 
+            var prDescription = PullRequestHelper.GetDescriptionForCreatingPullRequest(newBranchName);
```

Comment:
When this tool sends out a PR automatically it will now start having a "description" of steps that can be used. For example:

![image](https://user-images.githubusercontent.com/6559784/92949958-c598c900-f410-11ea-9a45-9e827400b87e.png)